recipe/meta.yaml

```diff
@@ -64,6 +63,7 @@ requirements:
     - conda-content-trust >=0.1.1
     - conda-env >=2.6
     - cytoolz >=0.8.1
+    - conda-libmamba-solver >=22.11.0
```

Comment:
We've just noticed the version is 22.12.0 - since there should never be a 22.11.0 ?? but we will fix it anyway.

Reply:
Could someone please submit a PR to update this here? Will also need a repodata patch

Reply:
I'll do the PR

Reply:
xref: https://github.com/conda-forge/conda-feedstock/pull/196

Reply:
https://github.com/conda-forge/conda-feedstock/pull/196